citations-api/api/pom.xml

```diff
@@ -40,6 +40,21 @@
       <artifactId>servlet-api</artifactId>
     </dependency>
     <dependency>
+        <groupId>org.json</groupId>
```

Comment:
Shouldn't have these libraries in the API, this means they end up getting deployed to /shared/lib.